packages/webpack/src/config/client.js

```diff
@@ -74,10 +74,6 @@ export default class WebpackClientConfig extends WebpackBaseConfig {
 
     if (!this.dev && cacheGroups.default && cacheGroups.default.name === undefined) {
       cacheGroups.default.name = (_module, chunks) => {
-        // Use default name for single chunks
-        if (chunks.length === 1) {
-          return chunks[0].name || ''
-        }
```

Comment:
This might change behavior, because it will now be returned as commons/name instead of just the name without the folder prefix for chunks with size 1. Is this intended?

Reply:
476af4304a6a6624d71b14d27183e5a7a9f59ef2